src/onnx/onnx.cpp

```diff
@@ -1151,6 +1200,8 @@ struct onnx_parser
         if(contains(info.attributes, "mode"))
```

Comment:
The mode attribute should be moved to before if (args.size() == 3) since the input value is used only for the constant mode.

Reply:
fixed